### ISSUE TITLE
Not so much an issue, but am curious if anyone else with CentOS has had similar problems CentOS 6, 7, 8, others. Python 3 from source and from installer.

### ISSUE THREAD

**Issue by sigkill** [3rdIteration/btcrecover#227]
While running CentOS 6, 7, and 8, the btcrecover.py output does not report back any status, until it completes.

This behavior is different than any Apple, or Debian based Linux I've run btcrecover on.

I suspect something to do with curses / ncurses, but I'm not sure. This appears to be pretty universal across the board, but I haven't tried from the physical console as of yet.

Just putting the question out there to see if anyone else has this behavior.

**Comment by 3rdIteration** [comment]
I have never tested against CentOS in any comprehensive way. Is there a specific version you are asking about? (Linux vs Stream, so Centos 8 stream?)

**Comment by sigkill** [comment]
Yes - am running CentOS 7 and a few other CentOS like. It's quite old, but am working on porting tools to the Xeon Phi 5110p

 lsb_release  -a
LSB Version:	:core-4.1-amd64:core-4.1-noarch
Distributor ID:	CentOS
Description:	CentOS Linux release 7.9.2009 (Core)
Release:	7.9.2009
Codename:	Core

This is the main node.

The other node runs an Intel specific version:
Linux xxxxx-mic0 2.6.38.8+mpss3.8.6 #1 SMP Thu Jul 25 13:16:12 EDT 2019 k1om GNU/Linux

**Comment by sigkill** [comment]
OTS: For BIP38 - there is a really fast 1040 keys per second repo on here too - https://github.com/agurodriguez/brute_force_bip38.git I think it's considerably faster than btcrecover, but it's also written in cpp, and is full of bugs.

**Comment by sigkill** [comment]
Okay I think I figured it out with the help from a friend. I suspect it's the readline library. I've installed on centos - the readline-devel and other readline packages, also the newest version of bash has a flag for enabling there as well.

My present system however, the only change was the addition of readline libraries with `yum install readline-static.x86_64 readline.x86_64 readline-devel.x86_64` 

This is a super old system, but the other system also has a bastardization of Intel / busybox / other libraries. Will check out compiling readline and installing there next.

I think this is an issue that isn't so much btcrecover.py, but an issue with CentOS, and others based off RedHat. Might be good info for the README or something. :)
